packages/http/src/resources/Hotspots.ts

```diff
@@ -34,19 +35,22 @@ export default class Hotspots {
     params: ListParams = {},
   ): Promise<{ hotspots: HTTPHotspotObject[]; cursor?: string }> {
     let url = '/hotspots'
+    const clientParams:{cursor?:string, search?:string} = { cursor: params.cursor }
```

Comment:
hm why does `clientParams` have `search` as an optional attribute, but doesn't populate it?

Reply:
If it's present, it gets populated on line 51